src/RenderWebGL.js

```diff
@@ -615,9 +621,30 @@ class RenderWebGL extends EventEmitter {
             return false;
         }
 
+        measure.start('newWay');
+
+        const point = twgl.v3.create(0, 0);
+        const iterator = bounds.points(point);
+        const drawable = this._allDrawables[drawableID];
+        const candidates = candidateIDs.map(id => this._allDrawables[id]);
+        let newWay = false;
+        while (iterator.next().value) {
```

Comment:
yes, looping over EVERY pixel and asking the Silhouette is still much faster than reading those pixels back from the GPU

Reply:
If anyone has any thoughts on an even more performant approach - please chime in!